src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx

```diff
@@ -366,8 +366,9 @@ const PopularRepositoriesTable = () => {
             <InputGroupItem isFill>
               <TextInput
                 isDisabled={isLoading}
+                type='text'
                 id='search'
-                ouiaId='filter_search'
+                ouiaId='popular_filter_search'
```

Comment:
I add this back as QE uses this to target the input for tests. 
This is likely the reason that your CI-CD Build was failing.

Reply:
Okay